BREAKING.md

```diff
@@ -68,6 +69,9 @@ This impacts the following public apis:
 
  In general the impact of these changes should be transparent. If you see compile errors related to Fluid object provider types with the above apis, you should transition those usages to [FluidObject](https://github.com/microsoft/FluidFramework/blob/main/common/lib/core-interfaces/src/provider.ts#L61) which is the replacement for the deprecated IFluidObject.
 
+### Removing snapshot API from container runtime
```

Comment:
This should say `IRuntime`. It's already removed from container runtime right.

Reply:
We need this removed from `IRuntime` so we can get rid of the container runtime `snapshot` function. Small piece left; we are almost there.